modules/programs/zsh/default.nix

```diff
@@ -391,7 +401,24 @@ in
                   - config.xdg.dataHome (XDG data directory)
                   - config.xdg.cacheHome (XDG cache directory)
                 ''
-              ];
+              ]
+            ++
+              lib.optionals
+                (
+                  config.xdg.enable
+                  && !lib.versionAtLeast config.home.stateVersion "26.05"
+                  && options.programs.zsh.dotDir.highestPrio >= 1500
```

Comment:
TIL `isDefined` does not do what I thought it does. So there's no (good) way to check that a value is explicitly given in the module system and not through `default`, other than using the priority.

I guess that makes sense.

Reply:
Arf, can't close this comment, but this is not actionable, merely an observation.